### ISSUE TITLE
Cant install ms store or .msixbundle apps

### ISSUE THREAD

**Issue by Supply12** [maharmstone/quibble#123]
I have windows 10 build 19045 on winbtrfs
When i try to install uwp app it gives an error:
`Windows не удается выполнить развертывание в папку 32669SamG.ModernFlyouts_0.9.3.0_neutral_split.scale-100_pcy8vm99wrpcg файловой системы с типом Btrfs. (0x80073cfd)`

**Comment by predefine** [comment]
this is winbtrfs related problem

**Comment by heipiao233** [comment]
It's Microsoft's restriction.

**Comment by maharmstone** [comment]
Yes, I'm not sure that this is something that can be worked round.

**Comment by kristibektashi** [comment]
This isn't a quibble problem so it should be marked as wontfix and mentioned in the readme. Worth noting that already installed apps will probably still work fine, you just can't install new ones

**Comment by heipiao233** [comment]
I guess we can try to use a vhd with NTFS to deal with that?

**Comment by kristibektashi** [comment]
> I guess we can try to use a vhd with NTFS to deal with that?

You probably could

**Comment by heipiao233** [comment]
> > I guess we can try to use a vhd with NTFS to deal with that?
> 
> You probably could

However my quibble installation is corrupt, I can't test this now.

Update: I fixed it by compiling and using the latest quibble, however a BSOD happens when I attach a vhd (#128).
